ios-swiftui/realm-template-apps-swiftui/Views/TodosView.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+import RealmSwift
+
+/// Use views to see a list of all Todos, add or delete todos, or logout.
+struct TodosView: View {
+    var leadingBarButton: AnyView?
+    // ObservedResults is a mutable collection; here it's
+    // all of the Todo objects in the realm.
+    // You can append or delete todos directly from the collection.
+    @ObservedResults(Todo.self) var todo
+    @State var addTodo = false
+    @State var todoSummary = ""
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                if addTodo {
+                    NewTodo(addTodo: $addTodo)
```

Comment:
I think even "NewTodo" might not be as effective of a name for a view. It's more of a "NewTodoInputPrompt" or Modal or something.

Reply:
Fair point. I wonder if "InputPrompt" is too specific, though - the view creates a new Todo object, lets the user input a summary, and then adds the object to the realm via an implicit write. I've added comments to go into more detail about what the view does, and am renaming it to `CreateTodo` - hopefully that's clear enough.

Reply:
Personally I wouldn't use a verb phrase as a name of this kind of class. "Create Todo... View"? If I did see a class called CreateTodo I would think it was some kind of model class that represented the payload of creating a Todo or maybe it was a special kind of Todo model.